src/StatusQ/Components/StatusChatListCategoryItem.qml

```diff
@@ -14,6 +14,12 @@ StatusListItem {
     leftPadding: 8
     rightPadding: 8
 
+    property StatusChatListCategoryItemTooltipSettings tooltips: StatusChatListCategoryItemTooltipSettings {
```

Comment:
We don't need to expose them here because we already expose `addButton` and `menuButton`.

As mentioned in #226 this is about **`StatusChatListAndCategories`**. Please add an API to configure tooltips there.